lib/run_loop/detect_aut/xcode.rb

```diff
@@ -39,7 +41,19 @@ def find_xcodeproj
       end
 
       # @!visibility private
-      def ignore_xcodeproj?(path)
+      def self.find_user_state_files
+        username = Etc.getpwuid.name
+        user_state_file_paths = []
+        Dir.glob("#{Dir.pwd}/**/xcuserdata/#{username}.xcuserdatad/UserInterfaceState.xcuserstate").each do |path|
```

Comment:
Just call `xcodeproj` to get the correct xcode project.

Then find the the `xcuserstate` directory from that xcode project.